djangocms_versioning/admin.py

```diff
@@ -816,6 +815,15 @@ def changelist_view(self, request, extra_context=None):
             )
         except (TypeError, ValueError):
```

Comment:
Do we know in what circumstances the following are caught: TypeError, ValueError? Comments would have helped us here

Reply:
It's basically because of `int()` and `.GET.get()`

Reply:
Do you want a comment for this here?